src/seeds.forum.cpp

```diff
@@ -532,6 +532,11 @@ ACTION forum::testsize (name id, uint64_t size) {
     size_set(id, size);
 }
 
+ACTION forum::testrnk (uint64_t rnk) {
```

Comment:
generally any action even a test action needs to have authorization set @juliolrmonteiro

Reply:
Just noting because I sometimes forget it too